lib/tree.rb

```diff
@@ -1,38 +1,58 @@
+require 'pry'
+
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_accessor :height, :age, :apples, :alive
```

Comment:
We want the tree to be the only thing that manages it's instance variables. Therefore, allowing accessors here gives the consumer of this class more information than they need.